components/match2/commons/match_group_input.lua

```diff
@@ -583,7 +654,161 @@ function MatchGroupInput.readMvp(match)
 	return {players = parsedPlayers, points = mvppoints}
 end
 
+---@param match table
+---@param opponentIndex integer
+---@param opponentInput table
+---@return {players: table[], substitutions: MatchGroupInputSubstituteInformation[]}
+function MatchGroupInput.extractManualPlayersInput(match, opponentIndex, opponentInput)
+	--players from manual input as `opponnetX_pY`
+	local players = Array.mapIndexes(function(playerIndex)
+		return Json.parseIfString(match['opponent' .. opponentIndex .. '_p' .. playerIndex])
+	end)
+	local manualInput = {players = players}
+
+	local substitutions, parseFailure = Json.parseStringified(opponentInput.substitutes)
+	if parseFailure then
+		substitutions = {}
+	end
+
+	---@param playerData table|string|nil
+	---@return standardPlayer?
+	local getStandardPlayer = function(playerData)
+		if not playerData then return end
+		playerData = type(playerData) == 'string' and {playerData} or playerData
+		local player = {
+			displayName = Logic.emptyOr(playerData.displayName, playerData.displayname, playerData[1] or playerData.name),
+			pageName = Logic.emptyOr(playerData.pageName, playerData.pagename, playerData.link),
+			flag = playerData.flag,
+			faction = playerData.faction or playerData.race,
+		}
+		if Logic.isEmpty(player.displayName) then return end
+		player = PlayerExt.populatePlayer(player)
```

Comment:
Does this need sync?

Reply:
this is a call to sync just without setting the wiki vars

do you mean you want it changed to sync?
or do you mean you want it removed?

Reply:
Should it have fetchPlayer = true?

Reply:
fetchPlayer is enabled by default
only valid value is `false`
https://github.com/Liquipedia/Lua-Modules/blob/f4d46aa8ad88d66a073b02013678f56164048916/components/opponent/commons/player_ext.lua#L175-L176